packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionProductBlock/WfoProductBlockKeyValueRow.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react';
+
+import { EuiBadge } from '@elastic/eui';
+
+import { useSubscriptionDetailValueOverride } from '@/components';
+import { useWithOrchestratorTheme } from '@/hooks';
+import { FieldValue } from '@/types';
+import { camelToHuman } from '@/utils';
+
+import { getStyles } from './styles';
+
+export type WfoProductBlockKeyValueRowProps = {
```

Comment:
No export needed here right?

Reply:
Exporting the props type to provide override possibilities by clients